src/it/tests/install-features-it/build.xml

```diff
@@ -1,33 +1,35 @@
-<?xml version="1.0"?>
-<project xmlns:wlp="antlib:net.wasdev.wlp.ant" name="net.wasdev.wlp.ant.it">
-
-    <path id="wlp-ant-tasks.classpath">
-        <fileset dir="${basedir}/../../../../target" includes="wlp-anttasks-*.jar" />
-    </path>
-    <typedef resource="net/wasdev/wlp/ant/antlib.xml" uri="antlib:net.wasdev.wlp.ant" classpathref="wlp-ant-tasks.classpath" />
-
-    <property name="target.dir" value="${basedir}/../install-server-it/target" />
-
-    <!-- Defining server configuration -->
-    <property name="wlp.install.dir" value="${target.dir}/wlp" />
-    <property name="wlp.usr.dir" value="${target.dir}/wlp_usr" />
-    <property name="wlp.output.dir" value="${target.dir}/wlp_output" />
-    <property name="servername" value="install.features" />
-
-    <target name="installFeatures">
-        <wlp:server id="testServer" installDir="${wlp.install.dir}" serverName="${servername}" userDir="${wlp.usr.dir}" outputDir="${wlp.output.dir}" operation="create" />
-        
-    	<wlp:install-feature ref="testServer" acceptLicense="true" name="jsonp-1.0" />
-    		
-    	<wlp:install-feature ref="testServer" acceptLicense="true">
-            <feature>jdbc-4.1</feature>
-        </wlp:install-feature>
-    	
-    	<wlp:uninstall-feature ref="testServer">
-    	    <feature>jsonp-1.0</feature>
-    	    <feature>jdbc-4.1</feature>
-    	</wlp:uninstall-feature>
-
-    </target>
-
-</project>
+<?xml version="1.0"?>
+<project xmlns:wlp="antlib:net.wasdev.wlp.ant" name="net.wasdev.wlp.ant.it">
+
+    <path id="wlp-ant-tasks.classpath">
+        <fileset dir="${basedir}/../../../../target" includes="wlp-anttasks-*.jar" />
+    </path>
+    <typedef resource="net/wasdev/wlp/ant/antlib.xml" uri="antlib:net.wasdev.wlp.ant" classpathref="wlp-ant-tasks.classpath" />
+
+    <property name="target.dir" value="${basedir}/../install-server-it/target" />
+
+    <!-- Defining server configuration -->
+    <property name="wlp.install.dir" value="${target.dir}/wlp" />
+    <property name="wlp.usr.dir" value="${target.dir}/wlp_usr" />
+    <property name="wlp.output.dir" value="${target.dir}/wlp_output" />
+    <property name="servername" value="install.features" />
+
+    <target name="installFeatures">
+        <wlp:server id="testServer" installDir="${wlp.install.dir}" serverName="${servername}" userDir="${wlp.usr.dir}" outputDir="${wlp.output.dir}" operation="create" />
+
+    	<wlp:install-feature ref="testServer" acceptLicense="true" name="jsonp-1.0" />
+    	
+        <wlp:install-feature ref="testServer" acceptLicense="true" name="jacc-1.5,jaspic-1.1" />
```

Comment:
Yes, I was trying to make sure it worked, but tried to choose as small of features as possible that could also be uninstalled.